conversions/octal_to_decimal.py

```diff
@@ -9,26 +9,40 @@ def oct_to_decimal(oct_string: str) -> int:
     >>> oct_to_decimal("-45")
     -37
     >>> oct_to_decimal("2-0Fm")
+    Traceback (most recent call last):
```

Comment:
These are optional and do not add much value

Reply:
I added them because the doctest failed without them.